.github/workflows/tags.yaml

```diff
@@ -10,11 +10,6 @@ jobs:
       - run: sudo apt-get update && sudo apt install -y libgpgme-dev libassuan-dev libdevmapper-dev btrfs-progs libbtrfs-dev
```

Comment:
just happened to see these. are all these packages still needed?

Reply:
Good catch, I completely overlooked that line. Will remove :)